profiles/dab_jwt/pulp_config.env

```diff
@@ -21,3 +21,6 @@ PULP_DYNACONF_AFTER_GET_HOOKS=["read_settings_from_cache_or_db", "alter_hostname
 
 # disable user/group modifications
 PULP_ALLOW_LOCAL_RESOURCE_MANAGEMENT=false
+
+# triggers role migrations ...
+DAB_RBAC_ENABLED="1"
```

Comment:
Does this need to be included in the installers' environment?

Reply:
No, we need to talk to QE about what marks they are or are intending to use for downstream pipelines.

Reply:
No, this is no longer necessary because the management command it triggered to run no longer exists.